tests/test_pip.py

```diff
@@ -33,11 +35,6 @@ def test_pip_exists(Package, AnsibleDefaults, AnsibleFacts):
         AnsibleFacts - Used to determine OS Distribution Version
```

Comment:
When the function arguments change, the `AnsibleFacts` docs here become obsolete.